<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnifferDialog</class>
 <widget class="QDialog" name="SnifferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Sniffers</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="targetEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="targetLabel">
     <property name="text">
      <string>Target</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QSpinBox" name="channelSpin">
     <property name="maximum">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="channelLabel">
     <property name="text">
      <string>Channel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close Window</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="snifferLabel">
     <property name="text">
      <string>Opened sniffers and files</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="typeCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableView" name="snifferTable">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>16</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
